src/config.rs

```diff
@@ -116,6 +118,12 @@ impl Config {
       .version_message("Print version information")
       .setting(AppSettings::ColoredHelp)
       .setting(AppSettings::TrailingVarArg)
+      .arg(
+        Arg::with_name(arg::CHECK_FMT)
+          .long("check")
+          .requires(cmd::FORMAT)
+          .help("Make `--fmt` check formatting only. Exits with 1 and prints diff if formatting is required."),
```

Comment:
I stole this from `rustfmt`'s help:

```suggestion
          .help("Run `--fmt` in 'check' mode. Exits with 0 if justfile is formatted correctly. Exits with 1 and prints a diff if formatting is required."),
```